src/mscorlib/src/System/Globalization/CalendarData.Windows.cs

```diff
@@ -123,13 +123,17 @@ internal static int GetCalendars(String localeName, bool useUserOverride, Calend
         {
             EnumCalendarsData data = new EnumCalendarsData();
             data.userOverride = 0;
+#if CORECLR
```

Comment:
Other places are using using under `#if CORECLR` at the top of the file to reduce number of ifdefs. Do the same here?